### PR TITLE
Add contributing guidelines and MAINTAINERS file.

### PR DESCRIPTION
**Pull request by alexeyklyukin**

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
👍

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Contributing guidelines
+======================
+
+Thank you for using acid-tools! We are happy to share this project with you. We admit our code is not perfect and can be improved. Perhaps you stubmled upon a bug in the code, or think about implementing a new feature? We are happy to get your feedback! Here's how you can contribute:
```
Comment (avaczi) [inline CONTRIBUTING.rst]: stumbled

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+------------
+
+If you find a bug - please, create an issue on github, describing the bug and how to reproduce it.
+
+Contribute code and documentation
+-----------------------------------
+
+1. Unless it's a trivial fix, please, create an issue describing what you want to do and how are you going to do it.
+2. Fork a project and implement your changes in a separate branch there.
+3. Make sure your changes are self-contained and as small as possible. It's better to contribute 10 small pull-request than one big one.
```
Comment (avaczi) [inline CONTRIBUTING.rst]: pull requests

File: MAINTAINERS [hunk 3]
```diff
@@ -0,0 +1 @@
+Stefan Litsche <stefan.listsche@zalando.de>
```
Comment (avaczi) [inline MAINTAINERS]: litsche@

**Comment by avaczi** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

File: MAINTAINERS (hunk 3 above)
Comment (slitsche) [inline MAINTAINERS]: Please s/listsche/litsche

**Pull request merged**